drivers/interrupt_controller/exti_stm32.c

```diff
@@ -342,13 +365,13 @@ static inline void __stm32_exti_isr_22(void *arg)
 	__stm32_exti_isr(22, 23, arg);
 }
 #endif /* CONFIG_SOC_SERIES_STM32{F4X F7X, F2X} */
+#endif /* CONFIG_SOC_SERIES_STM32{F4X F7X, F2X, G0X} */
```

Comment:
Please remove the comment.
We have sanitychecks to detect non existing CONFIG symbols and we clean them up periodically. Better not to add them in first place.
Apply everywhere.